.github/workflows/erlang.yml

```diff
@@ -11,25 +11,50 @@ jobs:
   ci:
     name: Run checks and tests over ${{matrix.otp_vsn}} and ${{matrix.os}}
     runs-on: ${{matrix.os}}
-    container:
-      image: erlang:${{matrix.otp_vsn}}
     strategy:
       matrix:
-        otp_vsn: ["19.0", "19.3",
-                  "20.0", "20.1.7", "20.3.8.22",
-                  "21.0.9", "21.1.4", "21.2.7", "21.3.8.1",
-                  "22.0.7", "22.2.8", "22.3.4",
-                  "23.0.2", "23.2"]
-        os: [ubuntu-latest]
+        otp: ["25.3", "26.1"]
+        rebar3: ['3.22.1']
+        os: [ubuntu-22.04]
```

Comment:
Not sure if this is related to the build failure or not, but I don't know if this works the way they want. They refer to `matrix.otp_vsn` but they renamed `otp_vsn` to just `otp`.